packages/mask/src/plugins/Wallet/SNSAdaptor/index.tsx

```diff
@@ -17,9 +16,9 @@ const sns: Plugin.SNSAdaptor.Definition = {
                 <ApplicationBoardDialog />
                 <ConnectWalletDialog />
                 <GasSettingDialog />
-                {getEnumAsArray(NetworkPluginID).map(({ key, value: pluginID }) => (
-                    <TransactionSnackbar key={key} pluginID={pluginID} />
-                ))}
+                {getEnumAsArray(NetworkPluginID).map(({ key, value: pluginID }) =>
+                    TransactionSnackbar.open({ pluginID }),
```

Comment:
this doesn't look right. open() looks like a side-effect function you should not call it in the render function. if open() does not have a side-effect, please rename that function.